canvas_modules/harness/package.json

```diff
@@ -21,6 +21,7 @@
     "express-session": "1.17.1",
     "isomorphic-fetch": "3.0.0",
     "istanbul-middleware": "0.2.2",
+    "js-file-download": "^0.4.12",
```

Comment:
We don't use the ^ syntax in our package.json since this will automatically import a new version at build time.